goss.yaml

```diff
@@ -0,0 +1,10 @@
+http:
+  http://localhost:8000:
+    title: Check for Welcome page
+    status: 200
+    allow-insecure: false
+    no-follow-redirects: false
+    timeout: 5000
+    body: ['Jarbas']
+
+##TODO add API checks
```

Comment:
I rather open an issue than leaving `TODO` in the source, but that's OK ; )

I think more important than checking the API (if home screen is working probably the rest of the work Django does is also OK) is to check static files (served directly by nginx in production).

Reply:
I leave it there to remind me of that. We could remove it now and open an issue. I just don't have the time to see every API call and test it.

Reply:
Forgot to mention the size reduction. From 373M to 178M

Reply:
Yep, I'd remove and open an issue… you can start with _When #285 is merged we need to add more Goss tests etc etc_. Then people can help you listing interesting stuff to test and you don't have to list everything yourself ; )